POSTER_PROJECTS/README.md

```diff
@@ -1,52 +1,143 @@
+- Activity: These sessions consist on gathering people into small groups (2~4) to explore and understand in depth one of the selected topics. At the end of the session, everyone should have a poster ready (this will be fun!). At the end of each day, all of the poster groups will give a one-minute presentation of their poster.
+- Outcome: The posters and discussions will then be converted into docs and/or specs to the IPFS repos.
+
+### How
+
+Have fun working in small groups to create posters and other documentation, then give a one-minute presentation of your work.
+
+- Purpose: Get people to learn some part of the code base in depth and through that process of learning, create materials that can be used by others to learn that part of the code base.
+- Length: 1:45h
+- Details: During this session, use your time to learn some part of the code base in depth together with a small team of collaborators. Together, create a poster that others can use to learn about that part of the code base.
+
+Some of the things to do during this session together with the other people on your poster team:
+
+- **Review any formal documentation** relevant to the topic you've chosen
+- **Find and read all the "pseudo-specs"** -- gather and read through all the partial documentation that's floating around. Discuss it together.
+- **Look through the related code**
+- **Look for relevant reference materials**: papers, recordings of talks, etc.
+- **Clarify and test your understanding** by discussing all of this information
+- **Interact with experts in the room** - people like Tech Leads will be circulating around the groups
+- **Compile all of this information** into something that's easy to consume (a poster)
+- **Plan how this information can be incorporated** into the existing documentation and repositories.
+- **Choose who will present your poster.**
+- _If you have time, submit a PR with your new info into the corresponding repository._
+
+At the end of the day, each group will have 1 minute to present their poster. Make sure to choose in advance who will be presenting.
+
+## How you will choose which poster to work on
+
+Poster teams should have a maximum of 4 people and a minimum of 2. People will choose which poster to work on using this process:
+- For this event we have a [list of topics](#topics) below.
+- The meeting organizers will draw a spot on the ground for each of the poster topics.
+- Before the start of the session, everyone must stand on the poster topic they want to work on. If more than five people try to join a topic, you must re-balance people to other topics.
+- When everyone has chosen a topic and no topics have more than five people, write down the list of participants for that topic.
+- Take a flip chart, markers, etc and work wherever you want to work.
+
+## Topics
+
+| # | Topic | Description |
+| :---: | --- | --- |
+| 1 | UnixfsV1 | [unixfsv1.md](./TOPICS/unixfsv1.md) |
```

Comment:
#tip: avoid creating another layer of indirection. Just do `topic-<name>` if you want to differentiate the README from the rest.